src/openrct2/drawing/Text.cpp

```diff
@@ -150,6 +150,15 @@ void gfx_draw_string(rct_drawpixelinfo* dpi, const_utf8string buffer, uint8_t co
     DrawText(dpi, x, y, &_legacyPaint, buffer);
 }
```

Comment:
Is the old function still used? If not, please delete it.

If it is, make it call the new one, so we don't duplicate code (and can follow-up on its removal on another PR to not increase the size of this diff)